hnn_core/optimization/optimizer.py

```diff
@@ -0,0 +1,382 @@
+"""Parameter optimization."""
+
+# Authors: Carolina Fernandez <cxf418@miami.edu>
+#          Nick Tolley <nicholas_tolley@brown.edu>
+#          Ryan Thorpe <ryan_thorpe@brown.edu>
+#          Mainak Jas <mjas@mgh.harvard.edu>
+
+import numpy as np
+
+from optimization import _rmse_evoked
```

Comment:
```suggestion
from .metrics import _rmse_evoked
```
For debugging purposes, let's be more explicit here so that we can more easily know which file `_rmse_evoked` comes from.